build.gradle

```diff
@@ -99,8 +100,14 @@ dependencies {
         exclude module: "logback-classic"
     }
 
+    // storage
+    compile 'org.springframework.boot:spring-boot-starter-jdbc:1.3.1.RELEASE'
+    compile 'org.postgresql:postgresql:9.4.1207.jre7'
+
     // misc
     compile 'org.zalando.stups:stups-spring-oauth2-server:1.0.0'
+    compile 'org.zalando:jackson-datatype-problem:0.5.0'
```

Comment:
I've added this as the "howto" explained but it didn't solve the problem of serialization of `Optional`s. I need to investigate further.